src/generated/sync/META-INF/accesstransformergenerated.cfg

```diff
@@ -0,0 +1,377 @@
+# render state shard classes
```

Comment:
Add a few lines at the top saying that this file is autogenerated and should not be edited by hand. Instead, the rules inside of `build.gradle` should be edited before rerunning `generateAccessTransformers`.